code/style.css

```diff
@@ -1,7 +1,545 @@
+}
+
+h2 {
+  font-size: 2em;
+}
+
+h3 {
+  font-family: 'Bebas Neue', cursive;
+  letter-spacing: 1.7px;
+}
+
+.small-text {
+  font-size: 0.825rem;
+  font-weight: 500;
+}
+
+button,
+.modal-btn {
+  font-family: 'Bebas Neue', cursive;
+  letter-spacing: 1.6px;
+  font-size: 1em;
+  text-align: center;
+  text-transform: uppercase;
+}
+
+.infotext p {
+  font-family: 'Bebas Neue', cursive;
+  font-size: clamp(1.125em, 1.5vw, 1.4em);
+  text-align: center;
+  line-height: 1.6em;
+  letter-spacing: 1.5px;
+}
+
+.accordion h3,
+.accordion p {
+  color: #222222;
+}
+
+/* ----------- BUTTONS AND LINKS ----------- */
+
+.btn, 
+button {
+  display: block;
+  position: relative;
+  padding: 0;
+  margin-top: 1em;
+  color: #e9e9e9;
+  text-decoration: none;
+  overflow: hidden;
+  width: 15em;
+}
+
+
+.btn-gradient,
+a {
+  background-image: linear-gradient(to right, #2C2A2A 0%, #312f30 51%, #555253 100%);
+  background-size: 200% auto;
+  padding: 12px 32px;
+  transition: 0.5s;
+  color: #e9e9e9;
+  border: none;
+  border-radius: 5px;
+  display: block;
+  overflow: hidden;
+}
+
+.btn-gradient:hover {
+  background-position: right center; /* change the direction of the change here */
+}
+
+@media (min-width:992px) {
+  button,
+  .btn {
+    font-size: 1.2em;
+  } 
+  
+  .btn::after,
+  button::after {
+    content: "";
+    position: absolute;
+    width: 80%;
+    height: 0.2em;
+    align-self: center;
+    background-color: #ab8292;
+    opacity: 1; 
+    transition: opacity 300ms, transform 300ms;
+    transform: translate3d(150%, 0, 0);
+  }
+  
+  button::after {
+    bottom: 0.4em;
+    width: 70%;
+    left: 2.2em;
+  }
+  
+  .btn::after {
+    bottom: 0.6em;
+    left: 1.45em;
+  }
+  
+  .btn:hover::after,
+  .btn:focus::after,
+  button:hover::after {
+    transform: translate3d(0, 0, 0);
+  }
+}
+
+
+/* ----------- HEADER ----------- */
+
+header {
+  background-image: url("./images/hero-image2.png");
+  background-repeat: no-repeat;
+  background-size: cover;
+  object-fit: contain;
+  max-width: 100vw;
+  min-height: 75vh; /*change viewheight??*/
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+/* ----------- NAVBAR ----------- */
+
+ul {
+  display: none;
+}
+
+.nav-hamburger {
+  display: block;
+  font-size: 1.5rem;
+  color: #e9e9e9;
+  position: fixed;
+  right: 0.15em;
+  top: 0.15em;
+  cursor: pointer;
+}
+
+@media (min-width:667px) {
+  .nav-hamburger {
+    display: block;
+    font-size: 2.5rem;
+    color: #222222;
+    position: fixed;
+    right: 0.25em;
+    top: 0.25em;
+    width: 1em;
+  }
+  
+}
+
+@media (min-width:992px) {
+  
+  .nav-hamburger {
+    display: none;
+  }
+
+  ul {
+    display: block;
+  }
+
+  nav {
+    display: block;
+    background: rgba(0, 0, 0, 0.5);
+    position: fixed; 
+    right: 0;
+    top: 0;
+    width: 6em;
+  }
+
+  nav ul {
+    margin: 0;
+    padding: 0.5em 0;
+    list-style: none;
+    }
+    
+  nav ul li a {
```

Comment:
Try naming <a class="nav-link>, so you can reuse the code later. As using many sub-tag, or sub-class may be confusing for the one who reads your code

Reply:
I really needed this tip. For some reason I messed up with my a-tags and couldn´t find a good way out of it. Your tip will definitely help 👍